### PR TITLE
[GitHub Actions] added missing checks for builds to build successful

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, the reason for the PR| There were missing checks in the build successful.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

----
:globe_with_meridians: Live Preview:
  - https://tl-add-missing-checks-to-build-successful.odin.shopsys.cloud
  - https://cz.tl-add-missing-checks-to-build-successful.odin.shopsys.cloud

### CODE REVIEW

File: .github/workflows/docker-build.yaml [hunk 1]
```diff
@@ -789,12 +789,15 @@ jobs:
             STANDARDS_STOREFRONT_RESULT: ${{ needs.standards-storefront.result }}
             TRANSLATIONS_DUMP_RESULT: ${{ needs.translations-dump-check.result }}
             REVIEW_RESULT: ${{ needs.review.result }}
+            CHECK_CONSOLE_COMMANDS_RESULT: ${{ needs.check-console-commands.result }}
+            TESTS_STOREFRONT_ACCEPTANCE_RESULT: ${{ needs.tests-storefront-acceptance.result }}
+            UNIT_TESTS_STOREFRONT_RESULT: ${{ needs.unit-test-storefront.result }}
```
Comment (vitek-rostislav) [inline .github/workflows/docker-build.yaml:794]: should not these three jobs be added to the `needs` section of the `build-successful` job?

**Review by vitek-rostislav — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-build.yaml (hunk 1 above)
Comment (TomasLudvik) [reply .github/workflows/docker-build.yaml:794]: You are correct! I have added them and squashed it.

**Comment by vitek-rostislav** [comment]
The build is strangely failing on migrations :slightly_frowning_face:

**Comment by vitek-rostislav** [comment]
The build is strangely failing on migrations :slightly_frowning_face:

**Review by vitek-rostislav — APPROVED** [review]

**Pull request merged**